src/controllers/user.controllers.js

```diff
@@ -0,0 +1,41 @@
+import User from '../models/user';
+import { registerValidation, loginValidation } from '../routes/user/validation';
+import { StatusCodes } from 'http-status-codes';
+import bcrypt from 'bcryptjs';
+
+export const createUser = async (data, response) => {
```

Comment:
The only params passed to this function should be ones required to userEdit operation, then the results should be returned and response handled in routes.